client/src/components/UserForm.module.css

```diff
@@ -7,4 +7,20 @@
   display: flex;
   justify-content: space-between;
   margin-bottom: 25px;
+}
+button[type="submit"] {
```

Comment:
I'd say this is a bit generic and makes the component code hard to read, I'd suggest using the `mt-x` class from bootstrap.

https://getbootstrap.com/docs/4.0/layout/utilities-for-layout/#margin-and-padding